common/util/error_type.go

```diff
@@ -61,7 +62,7 @@ func errorType(err error) string {
 		q = q[:len(q)-1]
 		errType := fmt.Sprintf("%T", err)
 		if !wrapperErrorTypes[errType] {
-			return errType
+			return strings.TrimPrefix(errType, "*")
```

Comment:
I see that this code existed already, but it's not clear to me why an error would start with one or multiple `*`? If this is sth we introduced, could this be a constant in a single place?

Reply:
`*fmt.wrapError` is the type of an error. So is `*serviceerror.Foo`.

We don't care that they're pointers, so we remove the prefix